### ISSUE TITLE
Dummy rundll32 process gnaws my CPU

### ISSUE THREAD

**Issue by MuscularPuky** [ALTaleX531/TranslucentFlyouts#107]
When startup, TransparentFlyouts makes a rundll32 process, and it starts to eat the CPU about 10~20%.
Even I end that process, TransparentFlyouts does not terminate. it's still there.
[rundll32.zip](https://github.com/ALTaleX531/TranslucentFlyouts/files/14494849/rundll32.zip)

**Comment by MuscularPuky** [comment]
it's 3.0

**Comment by ALTaleX531** [comment]
If you're talking about TranslucentFlyouts.Immersive, then the issue won't be resolved for a while because I need to refactor its code, which takes time.

**Comment by MuscularPuky** [comment]
ah is this cause of immersive? there's 2 rundll32s

**Comment by ALTaleX531** [comment]
Yes, after looking at the memory dump file you provided, this was identified as an issue with TranslucentFlyouts.Immersive and I'll try to improve this.
